### PR TITLE
If a document is created within a collection, publish it automatically.

### PR DESCRIPTION
**Pull request by feingold-ant**
If a document is created within a collection, publish it automatically. Drafts can still be created outside of collections (and subsequently added to a collection when published).

### CODE REVIEW

File: server/commands/documentCreator.ts [hunk 1]
```diff
@@ -147,6 +147,10 @@ export default async function documentCreator({
     }
   );
 
+  if (collectionId) {
+    publish = true;
+  }
```
Comment (bkrausz) [inline server/commands/documentCreator.ts:152]: Should we do this on the client instead? Silently turning non-publish into publish in the app may be a bit subtle.

Comment (tommoor) [reply server/commands/documentCreator.ts:152]: Yea agreed, it would be nice to keep the API backwards compatible as well

Comment (feingold-ant) [reply server/commands/documentCreator.ts:152]: Sure thing

File: app/menus/NewDocumentMenu.tsx [hunk 2]
```diff
@@ -26,7 +26,7 @@ function NewDocumentMenu() {
       placement="bottom"
     >
       <Button as={Link} to={newDocumentPath()} icon={<PlusIcon />}>
-        {t("New doc")}
+        {t("New draft")}
```
Comment (feingold-ant) [inline app/menus/NewDocumentMenu.tsx:29]: @tommoor should translation have been updated here?

Comment (tommoor) [reply app/menus/NewDocumentMenu.tsx:29]: It will be displayed on the home screen in top right, might be a bit confusing… I would leave as-is for now

**Review by tommoor — APPROVED** [review]

**Pull request merged**